### DISCUSSION TITLE
Moving save file from mGBA to open_agb_firm?

### DISCUSSION THREAD

**Discussion by TehLinkz** [profi200/open_agb_firm#107] [General]
Is it possible to move a save file from mGBA to open_agb_firm? I tried putting my .sav file from mGBA into /3ds/open_agb_firm/saves but that didn't work.

Any ideas?

**Comment by dLeandroT** [comment]
Yes, that is possible.

Check that the save file has exactly the same name as the game file.

**Reply by TehLinkz** [reply]
I confirmed it's the same, but no dice.

**Reply by Arkham-Slayer** [reply]
Patch the save file (not the rom) with this website https://exelotl.github.io/gba-eeprom-save-fix/

**Reply by TehLinkz** [reply]
> Patch the save file (not the rom) with this website https://exelotl.github.io/gba-eeprom-save-fix/

You are a legend, this was it, thank you so much!

**Reply by TehLinkz** [reply]
@Arkham-Slayer Do you know if I have to byte swap back if I ever wanted to take my save back from open_agb_firm to mGBA?

**Reply by Arkham-Slayer** [reply]
No, I have never tried it.

**Reply by profi200** [reply]
You need to convert it back for most other emulators to work.

**Reply by TehLinkz** [reply]
> You need to convert it back for most other emulators to work.

But if I use the eeprom save fix, its only one direction, correct? Because I was doing testing to see if I could get my save back off open_agb_firm, so i tried using the link again but the .sav wasn't recognized by other emulators.

**Reply by profi200** [reply]
I don't know what the site linked above does exactly but if done right you should get a bit perfect copy of the savegame if you run it through the converter again.

Note that games like to delete the save when they detect something isn't right. Make sure to only use a known working backup.
